### ISSUE TITLE
possible way for installing plugins

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#428]
*Issue migrated from trac ticket [#144](http://colloquy.info/project/ticket/144)*

**type:** Enhancement | **component:** Colloquy (Mac) | **severity:** Normal | **priority:** Low | **version:** 2.0 (Mac) | **status:** closed | **resolution:** Won't Fix | **keywords:** plugin
___
#### 2005-04-25 00:13:29: @akempgen created the issue

just an idea, but probably not worth it:
(sorry for the q'n'd copy&paste ;) )

> why not use one of these colloquy://www.colloquy.info/extras/myplugin.scpt links
> on the extras page
> link will be opened with colloquy and it can put it right where it wants it

> or use some kind of container file
> myscript.colloquy
> c opens it and puts the script within in the plugins folder
___
#### 2005-04-25 00:30:06: anonymous commented

Basically the script install process could be made a lot easier

___
#### 2006-08-21 03:49:17: @xeenon changed status from *new* to *closed*

___
#### 2006-08-21 03:49:17: @xeenon changed resolution from ** to *wontfix*

___
#### 2006-08-21 03:49:17: @xeenon commented

Same as #309, which is closed.
